src/app/downloads/downloads-table.component.ts

```diff
@@ -0,0 +1,135 @@
+import { Component, OnDestroy } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { Subscription } from 'rxjs/Subscription';
+import { EpisodeMetricsModel, EpisodeModel, FilterModel, INTERVAL_DAILY, INTERVAL_HOURLY, INTERVAL_15MIN } from '../ngrx/model';
+import { selectEpisodes, selectFilter, selectEpisodeMetrics } from '../ngrx/reducers';
+import { filterPodcastMetrics, filterAllPodcastEpisodes, filterEpisodeMetrics, metricsData, getTotal } from '../shared/util/metrics.util';
+import { mapMetricsToTimeseriesData, dayMonthDate, hourlyDateFormat } from '../shared/util/chart.util';
+import * as moment from 'moment';
+
+@Component({
+  selector: 'metrics-downloads-table',
+  template: `
+    <table *ngIf="episodeTableData && episodeTableData.length">
+      <thead>
+        <tr>
+          <th>Episode</th>
+          <th>Release Date</th>
+          <th>Total for period</th>
+          <th *ngFor="let date of dateRange">{{date}}</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr *ngFor="let episode of episodeTableData">
+          <td>{{episode.title}}</td>
```

Comment:
Nice scrolling action. Any chance we can make this part of the table "sticky"? Andrew suggested we make these headings always visible when the table is horizontally long.

Reply:
good idea!